src/utils/date.js

```diff
@@ -11,6 +11,11 @@ const dateFormatter = {
     return `${d.toLocaleDateString('en-US')} ${d.toLocaleTimeString('en-US')}`;
   },
 
+  parseTimestampFromSQLTimestamp(timestamp) {
+    const d = new Date(timestamp);
+    return `${d.toLocaleDateString('en-US')} ${d.toLocaleTimeString('en-US')}`;
```

Comment:
For now, to maintain consistency, I think so, because we use the same default on `parseTimestamp` function. But I agree this is not the best UX and we should change this on all experiences inside explorer.